CONFIGURATION.md

```diff
@@ -27,7 +27,7 @@ socket.receive.buffer.bytes              |  *  | 0 .. 100000000  |             0
 socket.keepalive.enable                  |  *  | true, false     |         false | Enable TCP keep-alives (SO_KEEPALIVE) on broker sockets <br>*Type: boolean*
 socket.nagle.disable                     |  *  | true, false     |         false | Disable the Nagle algorithm (TCP_NODELAY). <br>*Type: boolean*
 socket.max.fails                         |  *  | 0 .. 1000000    |             3 | Disconnect from broker when this number of send failures (e.g., timed out requests) is reached. Disable with 0. NOTE: The connection is automatically re-established. <br>*Type: integer*
-broker.address.ttl                       |  *  | 0 .. 86400000   |          1000 | How long to cache the broker address resolving results (milliseconds). <br>*Type: integer*
+broker.address.ttl                       |  *  | 0 .. 86400000   |         60000 | How long to cache the broker address resolving results (milliseconds). <br>*Type: integer*
```

Comment:
this file was not re-generated after you changed the default back so it still contains the original change.
Remove from PR